kpi-tracker/src/main.rs

```diff
@@ -899,7 +899,10 @@ async fn process_payday_block(
             let closed_pool_count = &closed_pool_count;
             async move {
                 let pi = node.get_pool_info(block_ident, baker_id).await?.response;
-                let pool_info = pi.pool_info;
+                let pool_info = pi
+                    .active_baker_pool_status
+                    .expect("pool info missing")
+                    .pool_info;
```

Comment:
I'm not really sure what else to do here - is there some kind of default value that can be used if the pool info is not present?

Reply:
I think this is probably fine. We are iterating over the active bakers in the `block_ident` block, so the pool info in that block must have an `active_baker_pool_status`. I think it would also be fine to just skip incrementing any counter if there's no `active_baker_pool_status`.

Reply:
Sounds reasonable to you, @limemloh?